content/SCALE/SCALECLIReference/Auth/CLISessions.md

```diff
@@ -12,4 +12,260 @@ tags:
 
 {{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
 
+## Sessions Commands
+
+The **sessions** namespace has seven commands and is based on functions found in the SCALE API and web UI. 
+It provides access to session information through the seven **sessions** commands. 
+
+You can enter commands from the main CLI prompt or the **sessions** namespace prompt.
+
+### Sessions Command
+
+The `sessions` command returns a table that displays each session id, whether or not each session is current and internal, each session origin, and the credentials and credential data used to log into each session.
+
+{{< expand "Displaying all sessions since system boot" "v" >}}
+#### Description
+
+The `sessions` command does not require entering properties or arguments.
+
```

Comment:
you can delete this empty line here in in line 32. And really all the commands. You can make these a paragraph.